File: ru.kmorozov.gbd.core/src/main/java/ru/kmorozov/gbd/core/logic/model/book/google/OnlineAccess.java
package ru.kmorozov.gbd.core.logic.model.book.google;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by km on 05.12.2015.
 */
class OnlineAccess implements Serializable {

    @SerializedName("url") private String url;
    @SerializedName("price") private String price;
    @SerializedName("list_price") private String listPrice;
}

File: ru.kmorozov.gbd.db/src/main/java/ru/kmorozov/db/core/logic/model/book/google/OnlineAccess.kt
package ru.kmorozov.db.core.logic.model.book.google

import com.google.gson.annotations.SerializedName

import java.io.Serializable

/**
 * Created by km on 05.12.2015.
 */
internal class OnlineAccess : Serializable {

    @SerializedName("url")
    private val url: String? = null
    @SerializedName("price")
    private val price: String? = null
    @SerializedName("list_price")
    private val listPrice: String? = null
}
